frontend/src/locales/scripts/en.json5

```diff
@@ -826,5 +826,43 @@
       source: "Open source site",
       creator: "Open creator page",
     },
+    resultCountLabel: {
+      creator: {
+        audio: {
+          zero: "No audio files by this creator in {source}",
+          count: "{count} audio file by this creator in {source}|{count} audio files by this creator in {source}",
+          countMore: "Over {count} audio files by this creator in {source}",
```

Comment:
These seem very similar, would it also be possible to interpolate the media type here too? Or is it just easier to define them separately?

Reply:
I think it's better to leave as much as possible fully written out because interpolation does not always work well for different languages, which might have very different pluralization and other grammar rules.